net.kirgroup.confy.json

```diff
@@ -0,0 +1,66 @@
+{
+    "app-id" : "net.kirgroup.confy",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.36",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "confy",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/debug",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "libhandy",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dprofiling=false",
+                "-Dintrospection=enabled",
+                "-Dgtk_doc=false",
+                "-Dtests=false",
+                "-Dexamples=false",
+                "-Dvapi=false",
+                "-Dglade_catalog=disabled"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/libhandy.git",
+                    "tag" : "1.0.3",
+                    "commit" : "7126d2e8da9dcdeb5751e60ff6a74930804f9637"
+                }
+            ]
+        },
+        "python3-icalendar.json",
+        {
+            "name" : "other-modules"
+        },
```

Comment:
```suggestion
```

Reply:
Nice, i didn't get from docs that I could leave that out 👍